js/views/modals/purchase/Purchase.js

```diff
@@ -162,7 +162,7 @@ export default class extends BaseModal {
       singleSelect: true,
       radioStyle: true,
       initialState: {
-        showOnlyCur: currencies[0],
+        showOnlyCur: '',
```

Comment:
This was only showing moderators that accept BTC, I assume it was there because originally we automatically selected the first currency. Since we changed it to require the user to select a currency (none are selected by default) this should also have no default.